addon/helpers/ember-sparkles/axis.js

```diff
@@ -10,8 +10,13 @@ export function emberSparklesAxis([ scale ], { position, tickFormat, ticks, widt
 
   let result = axisFn().scale(scale);
 
-  if (tickFormat) {
-    result.tickFormat(timeFormat(tickFormat));
+  switch(typeof tickFormat) {
```

Comment:
@taras  and I discussed - what do you think about the following: 
```
let formatter = typeof tickFormat === 'string' ? timeFormat(tickFormat) : tickFormat(timeFormat);
result.tickFormat(formatter);
```
it's a bit less verbose